src/main/java/org/neo4j/gis/spatial/DefaultLayer.java

```diff
@@ -19,10 +19,7 @@
  */
 package org.neo4j.gis.spatial;
 
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.Set;
+import java.util.*;
```

Comment:
In general, it's no good idea to import a whole package. The classes `Arrays`, `HashSet`, `Iterator`, `Iterable` and `Set` are far from everything that is to `java.util`, so you import tons of useless stuff which might even get a hindrance. For example, you might want to use a different `Date` class further on.